.github/actions/python-flake8-lint/action.yml

```diff
@@ -0,0 +1,22 @@
+# Define action name, description, and inputs
+name: "Python Flake8 Lint"
+description: "Lint python code with flake8"
+
+# Define action steps
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-python@v2
+      with:
+        python-version: "3.x"
+        architecture: x64
+
+    # E203: https://www.flake8rules.com/rules/E203.html - Whitespace before ':'
+    # E501: https://www.flake8rules.com/rules/E501.html - Line too long
+    # W503: https://www.flake8rules.com/rules/W503.html - Line break occurred before a binary operator
+    # W605: https://www.flake8rules.com/rules/W605.html - Invalid escape sequence
+    # E722: https://www.flake8rules.com/rules/E722.html - Do not use bare except, specify exception instead
+    - run: |
+        pip install flake8
+        flake8 --max-line-length 88 --ignore E203,E501,W503,W605,E722
+      shell: bash
```

Comment:
Let's split the job into separate "run" keys, there's not benefit in keeping this in one (unlike Dockerfile where each layer is cached and costs disk space) and it's easier to catch which step failed.